nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/test/java/org/apache/nifi/processors/kudu/MockPutKudu.java

```diff
@@ -61,7 +62,7 @@ protected Upsert upsertRecordToKudu(KuduTable kuduTable, Record record, List<Str
     }
 
     @Override
-    protected KuduClient createClient(final String masters) {
+    protected KuduClient createClient(final String masters, final KerberosCredentialsService credentialsService) {
```

Comment:
Could you add a test that covers the logic to create secure/unsecure KuduClients?